pages/security/csp.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Content Security Policy
+permalink: /security/content-security-policy
+layout: docs
+sidenav: security
+---
+
+# Content Security Policy
+
+## What is Content Security Policy?
+Content Security Policy (referred to as **CSP** in the rest of this guide) is a security measure designed by the W3C (World Wide Web Consortium) to mitigate the likelihood of Cross-Site Scripting (XSS) attacks and data injection. It is designed to be used in conjunction with other security practices currently recommended for web development.
+
+It can be enabled on either client- or server-side, and is unobtrusive in unsupported browsers — that is, browsers that don't support CSP will ignore its directives and load the page as normal.
+
+CSP is supported by the current versions of **all modern desktop browsers**: Safari, Chrome, Firefox, and IE Edge. It is also supported in recent versions iOS Safari and Chrome for Android.
+
+Unfortunately, support for numbered versions of IE is essentially zero, with no support for IE < 10 and only two directives supported in IE 10+.
+
+All of the above supported browsers support CSP 1.0, with 100% support for 2.0 in Webkit/Blink browsers, and partial, but very good, support in Firefox. CSP 2.0 has all the features of 1.0, with several additional directives and support for inline `<script>` and `<style>` tags (see [Caveats](#caveats)).
+
+## Usage
+CSP is straightforward to implement, and supports providing a policy via HTTP header or `<meta>` tag.
```

Comment:
I think it might be helpful to reword this as, "...supports providing a policy either server-side (HTTP header) or client-side (`<meta>` tag). We could link down the page to those two sections also.

Can we use the example on line 84, from the client-side implementation section, to demonstrate what the meta tag looks like? I feel like that would be particularly helpful to folks who are new to this.